Add basic explanation to ProgrammableAndroids section. (#106)
Slimefun/Wiki: pages/Normal-Androids.md

> A normal android is created as a base for a specialized android.

Reviewer: ```suggestion
A Normal Android is used as a base to create a specialized Android.
```

## Obtaining
Normal Androids can be crafted in an [Enhanced Crafting Table](https://github.com/Slimefun/Slimefun4/wiki/Enhanced-Crafting-Table).

## Usage

> Normal Androids can be programmed to only move.

Reviewer: ```suggestion
Normal Androids can only be programmed to move.
```

> To power Androids you need compatible fuel source on each tiers.

Reviewer: Same comment as on the previous ones regarding fuel

## Tiers

> There are three tiers on normal android, each with different modifiers listed below.

Reviewer: ```suggestion
There are three tiers on the Normal Android, each with different modifiers listed below.
```

> | Tier | Fuel Efficiency |
> | ---- | --------------- |
> | Basic | 1.0x |
> | Advanced | 1.5x |
> | Empowered | 3.0x |

Reviewer: ```suggestion
| Tier      | Fuel Efficiency |
| --------- | :-------------: |
| Basic     | 1.0x            |
| Advanced  | 1.5x            |
| Empowered | 3.0x            |
```

## Power Source

> Each tiers have their own fuel source listed below.

Reviewer: ```suggestion
Each tier requires different types of fuel, their possible fuels are listed below.
```

### Basic Tier

> | Item | Duration |
> | ---- | -------- |
> | Block of Coal | 1m 20s |
> | Dried Kelp Block | 1m 10s |
> | Blaze Rod | 45s |
> | Coal/Charcoal | 8s |
> | Any (Stripped) Wood/Log Type | 2s |
> | Any Planks Type | 1s |

Reviewer: ```suggestion
| Item             | Duration |
| ---------------- | -------- |
| Block of Coal    | 1m 20s   |
| Dried Kelp Block | 1m 10s   |
| Blaze Rod        | 45s      |
| Coal/Charcoal    | 8s       |
| Any Wood Log     | 2s       |
| Any Planks Type  | 1s       |
```

### Advanced Tier

> | Item | Duration |
> | ---- | -------- |
> | Bucket of Fuel | 8m 20s |
> | Bucket of Oil | 3m 20s |
> | Lava Bucket | 1m 40s ||

Reviewer: ```suggestion
| Item           | Duration |
| -------------- | -------- |
| Bucket of Fuel | 8m 20s   |
| Bucket of Oil  | 3m 20s   |
| Lava Bucket    | 1m 40s   |
```

### Empowered Tier

> | Item | Duration |
> | ---- | -------- |
> | Uranium | 41m 40s |
> | Neptunium | 20m 0s |
> | Boosted Uranium | 50m 0s |

Reviewer: ```suggestion
| Item            | Duration |
| --------------- | -------- |
| Uranium         | 41m 40s  |
| Neptunium       | 20m 0s   |
| Boosted Uranium | 50m 0s   |
```
